src/stylesheets/components/_alerts.scss

```diff
@@ -1,3 +1,17 @@
+// Alert variables ---------- //
+// scss-lint:disable TrailingWhitespace
```

Comment:
$usa-alerts variable for is giving me that specific lint error - I just feel that keeping this format is much more readable